### PR TITLE
Add ID and NAMES to sandboxes

### PR DESCRIPTION
**Pull request by runcom**
Split for the last commit from #30 

Address part of #31 - pods ids and names

### CODE REVIEW

**Comment by runcom** [comment]
we'll be talking more about the last commit in here after https://github.com/kubernetes-incubator/ocid/pull/30 and I figure out a way to have less dependencies :)

**Comment by runcom** [comment]
@mrunalp - alright, we're now using the old good ./hack/vendor.sh in the second commit here and the added deps are about 3k of code TOTAL, which means that commit removed about 10k lines of deps code.

Everything should be working fine now, you can review the first commit as is - which is ID and NAMES for sandboxes

**Comment by runcom** [comment]
DO NOT MERGE YET

**Comment by runcom** [comment]
@mrunalp everything's ok here, you can check the vendoring by cloning into an empty GOPATH and set it to `./vendor`

**Comment by runcom** [comment]
@mrunalp https://github.com/kubernetes-incubator/ocid/pull/34/commits/55c1d915185a9241de58f12c4db3ff080731b13d <- this is the commit related to the PR title though - rest is just vendoring

File: server/container.go [hunk 1]
```diff
@@ -16,11 +16,10 @@ import (
 // CreateContainer creates a new container in specified PodSandbox
 func (s *Server) CreateContainer(ctx context.Context, req *pb.CreateContainerRequest) (*pb.CreateContainerResponse, error) {
 	// The id of the PodSandbox
-	podSandboxID := req.GetPodSandboxId()
-	if !s.hasSandbox(podSandboxID) {
-		return nil, fmt.Errorf("the pod sandbox (%s) does not exist", podSandboxID)
+	podSandboxName := req.GetPodSandboxId()
```
Comment (mrunalp) [inline server/container.go]: The kubelet will be passing ID here that we return in status, not the name. So, we need to handle that. Basically, the name will be passed for pod/container on their creates, we return the IDs which is then used in subsequent calls.

Comment (runcom) [inline server/container.go]: Alright makes sense

Comment (runcom) [inline server/container.go]: @mrunalp Fixed

**Comment by mrunalp** [comment]
We should print the pod sandbox id on the client on pod create. 
Also, could you rebase so I can make sure this isn't breaking anything already fixed on master?

**Comment by runcom** [comment]
Ack to print the sandbox ID. 

I thought this was already rebased though. I'll rebase again.

**Comment by mrunalp** [comment]
Ran into some issues testing this yesterday. I will try it out again today from a clean env.

**Comment by runcom** [comment]
> Ran into some issues testing this yesterday. I will try it out again today from a clean env.

sure, let me know, I would write some tests after this is merged and #43 is merged also

**Comment by runcom** [comment]
@mrunalp rebased and fixed https://github.com/kubernetes-incubator/ocid/pull/34#issuecomment-248758321

**Comment by mrunalp** [comment]
Okay, works fine now :) LGTM

**Pull request merged**